opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/utils/MqttStatsPublisher.java

```diff
@@ -1136,6 +1136,10 @@ void populateApNodeMetrics(List<ServiceMetric> metricRecordList, Report report,
                     int nonWifi = (busy - (busyTx + busyRx)) / totalDurationMs;
                     checkIfOutOfBound("nonWifi", nonWifi, survey, totalDurationMs, busyTx, busyRx, busy, busySelf);
                     radioUtil.setNonWifi(nonWifi);
+                    
```

Comment:
Small comment for the old comment at 1099, it says "busyRx = 0; /* Rx = Rx_obss + Rx_errr (self and obss errors) */". I think it should say /* Rx = Rx_self + Rx_obss + Rx_errr (self, obss and self-errors) */